product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -76,7 +86,12 @@ to commit something. This approach requires application changes to use
 the CAMO transaction protocol to work correctly, as the application is in some way
 part of the consensus. For more on this approach, see [CAMO](camo).
 
-The `raft` decision uses PGDs built-in raft consensus for commit decisions. It can have low performance and is currently only required when using GROUP COMMIT with an ALL commit scope group.
+The `raft` decision uses PGDs built-in raft consensus for commit decisions. It
```

Comment:
@djw-m PGD's

Reply:
For Group commit, looking at the code this is what it is:
received: what you wrote is correct
replicated: ignored for group commit (intentional? @PJMODOS )
durable: confirms the transaction after the prepare phase has been flushed to disk
visible: Confirms the transaction after all of its changes are flushed to disk and it's visible to concurrent transactions